src/cecs277TermProject/RoachMotel.java

```diff
@@ -69,8 +70,26 @@ private String pay(PaymentStrategy paymentMethod, RoachColony colony, int numDay
 		Room paidRoom = occupiedRooms.get(colony);
 		// in progress, needs a way to calculate cost from amenities - tyler
 		double cost = paidRoom.getCost() * numDays;
+		logCheckOut(paymentMethod, colony, numDays, this.logger);
 		return paymentMethod.pay(cost);
 	}
+	
+	private void logCheckIn(RoachColony colony, Logger log)
+	{
+		Room loggedRoom = occupiedRooms.get(colony);
+		
+		log.log("Successfully Checked In: Colony " + colony + "checking into " + loggedRoom + "with amenities [PLACEHOLDER]");
```

Comment:
Use `String.format()` as it's more readable. Also, `loggedRoom` should include amenities (to-be-implemented by Gust), so you don't need to log them here.